File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        heap=[]
        import heapq
        for num in nums:
            heapq.heappush(heap,-(num))
        res=0
        for i in range(k):
            res=heapq.heappop(heap)
        return -(res)
        

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.cs
public class Solution {
    public int FindKthLargest(int[] nums, int k) {
        PriorityQueue<int,int> heap = new PriorityQueue<int,int>();
        foreach (int num in nums){
            heap.Enqueue(num,-num);
        }
        int res = 0;
        for (int i = 0; i < k; i ++){
            res = heap.Dequeue();
        }
        
        return res;
    }
}